packages/fetchai/connections/p2p_noise/aea/api.go

```diff
@@ -69,13 +71,24 @@ func (aea *AeaApi) Init() error {
 	if aea.sandbox {
 		return nil
 	}
+	env_file := os.Args[1]
+	fmt.Println("[aea-api  ][debug] env_file:", env_file)
 
 	// get config
+	err := godotenv.Load(env_file)
```

Comment:
Using an environment file avoids the issues of dynamically set environment variables not being available in subshells etc.